Makefile

```diff
@@ -18,12 +18,19 @@ docker-generate:
 	$(eval WORKDIR := /go/src/github.com/everoute/everoute)
 	docker run --rm -iu $$(id -u):$$(id -g) -w $(WORKDIR) -v $(CURDIR):$(WORKDIR) everoute/generate make generate
 
+images-test:
+	docker build -f build/images/release/Dockerfile -t harbor.smartx.com/everoute/dev .
+	docker push harbor.smartx.com/everoute/dev:latest
```

Comment:
nit:
`image-test` is unnecessary, and maybe it's not a graceful solution to use private registry here?

Reply:
These test related op will be removed.